SingularityService/src/main/java/com/hubspot/singularity/smtp/MailTemplateHelpers.java

```diff
@@ -34,6 +36,7 @@
   private static final String REQUEST_LINK_FORMAT = "%s/request/%s";
   private static final String LOG_LINK_FORMAT = "%s/task/%s/tail/%s";
   private static final String DEFAULT_TIMESTAMP_FORMAT = "MMM dd h:mm:ss a zzz";
+  private static final String DEFAULT_TASK_LOG_ERROR_REGEX = "ERROR";
```

Comment:
Going back and forth on if the default should be to search for errors in the logs, or if that's something the operator should need to actively configure. @tpetr any thoughts there on a sensible default behavior?